tests/std/tests/GH_000690_overaligned_function/test.cpp

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+// UNSUPPORTED: arm

```

Comment:
is this a new comment convention?

Reply:
This is a LIT comment. The lit test harness goes through comments and looks for magic comment lines like this one which tells it that this test is unsupported if the `arm` feature is available.